classes/RightSignature.php

```diff
@@ -77,9 +77,9 @@ public function prepackageTemplate($templateGuid, $callbackUrl=null)
 	 * @param int $documentGuid document GUID
 	 * @return RightSignature\Document
 	 */
-	public function document($documentGuid)
+	public function documentDetails($documentGuid)
```

Comment:
Judging by the following line, it doesn't look like it was ever implemented. So it's probably not worth leaving the old method hanging around.